libs/SmartSync/SmartSyncTests/MetadataManagerTests.m

```diff
@@ -44,12 +44,12 @@ @interface MetadataManagerTests : XCTestCase
 @implementation MetadataManagerTests
 
 static NSInteger const kRefreshInterval = 24 * 60 * 60 * 1000;
-static NSString* const kAccountOneId = @"001O000000m3UBiIAM";
-static NSString* const kAccountOneName = @"TEST ACCOUNT DO NOT TOUCH";
```

Comment:
What's wrong with these ones? They should be created and deleted by the test anyway.